### ISSUE TITLE
Accessibility Audit of Logs Drilldown App

### ISSUE THREAD

**Issue by yangkb09** [grafana/logs-drilldown#1878]
Conduct an accessibilty audit of Logs Drilldown to determine its WCAG level.

**Comment by yangkb09** [comment]
I did some initial research into conducting an accessibility audit on Logs Drilldown [GH issue]. The conclusion is that the Claude Code a11y-audit skill doesn’t produce a reliable enough audit that we can trust, so to conduct an a11y audit would require the traditional method of manual, screenreader, and automated (axe DevTools, not a CC skill) testing. @cakennedy and I discussed this and decided to pause the effort for now

If you’re interested in the details, here’s some more information:

- The a11y-audit CC skill produces a VPAT.md and accessibility-audit.md. The VPAT is the VPAT (i.e., the document needed to determine a WCAG level) and the accessibility-audit is a document containing all the issues CC found, categorized by severity.
- I manually tested all 6 critical level findings. My notes and corrections of CC are in the accessibility-audit doc linked above. If you just want to see the final set of verified findings, check out this doc
- My takeaways:
- ~Half of the critical findings had detail-level errors in things like where the bug lives, who owns it, and what the user-facing consequence actually is
- CC overestimates severity level (this is a semi-subjective thing)
- Overall, a11y-audit is able to detect potential issues, but human verification is needed
- WCAG level is hierarchical. To my understanding, the lowest level is 2.0 A. If you are partially or non-compliant with any of the “lower-tiered” criteria, you cannot claim a higher WCAG level

I’m going to reach out to @/Chris Bedwell to do some knowledge sharing when he’s back next week. Let me know if you have any questions! Happy to help you get started if you’d like to conduct an accessibility audit :slightly_smiling_face:
